gateway/api/user/user.go

```diff
@@ -95,16 +94,26 @@ func (u *Router) createUser(w http.ResponseWriter, r *http.Request) {
 		"email", user.GetEmail()))
 }
 
+type logininfo struct {
+	Email    string `json: "email"`
```

Comment:
I don't think you're supposed to have a space here:

```suggestion
	Email    string `json:"email"`
```

https://travis-ci.com/ubclaunchpad/pinpoint/jobs/180038401#L633